packages/browser/rollup.config.js

```diff
@@ -51,4 +51,13 @@ export default () => [
     external,
     plugins: [dts()],
   },
+  {
```

Comment:
They are intentionally not bundled but hand written. They expose a simple interface. File that exposes this type is not in dist.

Reply:
done